pom.xml

```diff
@@ -42,7 +42,6 @@
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
     <url>http://github.com/jenkinsci/${project.artifactId}-plugin</url>
-    <tag>display-url-api-2.3.3</tag>
```

Comment:
I think this should be `<tag>HEAD</tag>` for non-release commits, right?

Reply:
HEAD is default value :)

Reply:
Today I learned 😄